### DISCUSSION TITLE
code signature issue

### DISCUSSION THREAD

**Discussion by Evilfishbone** [snipem/gt7dashboard#33] [Q&A]
Got the app installed (hopefully correctly) for mac.  Ran the server, then when I tried to connect, I got these errors about file signature.  Anything I can do to get around it?  python 3.11

<img width="1032" alt="Screenshot 2024-09-19 at 11 14 54 AM" src="https://github.com/user-attachments/assets/fa0a422f-a0e6-48de-af7d-18ce623cac91">

**Comment by snipem** [comment]
Try again, I just removed the `salsa20` dependency. This might help your case.
